File: Python/bubble_sort.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n - i - 1):
            if arr[j] > arr[j + 1]:
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
    return arr


arr = [3, 7, 4, 8, 6, 2, 1, 5, 0, -5, -2]
print(bubble_sort(arr))

File: Kotlin/bubbleSort.kt
fun bubbleSort(arr: Array<Int>) {
    val n = arr.size
    for (i in 0 until n - 1) {
        for (j in 0 until n - i - 1) {
            if (arr[j] > arr[j + 1]) {
                val temp = arr[j]
                arr[j] = arr[j + 1]
                arr[j + 1] = temp
            }
        }
    }
}

fun main() {
    val arr = arrayOf(64, 34, 25, 12, 22, 11, 90, 0)
    bubbleSort(arr)
    println("Sorted array: ${arr.joinToString(", ")}")
}